### ISSUE TITLE
[a11y]: Missing Aria role cds-tab & cds-tabs, fails to meet accessibility standards

### ISSUE THREAD

**Issue by samlloyd-Dmtc** [carbon-design-system/carbon#23373]
### Package

@carbon/web-components

### Browser

Chrome

### Package version

v2.60.0

### Automated testing tool and ruleset

 Axe DevTools Extension 

### Description

Axe dev tools flags both cds-tab and cds-tabs for the following reasons. 
Cds tabs contains children (cds-tab) which have a non allowed role: [role=listitem]

<img width="596" height="268" alt="Image" src="https://github.com/user-attachments/assets/919090ad-dc3a-401a-81f1-dc084eda195a" />

Similarly cds-tab also flags the following violation:
Required ARIA parent role not present: list

<img width="980" height="368" alt="Image" src="https://github.com/user-attachments/assets/d41df146-0038-4ebd-a1bc-363b50c4d8e4" />

Both examples are visible on web componenets story book.

### Reproduction/example

https://web-components.carbondesignsystem.com/?path=/story/components-tabs--default

### Steps to reproduce

Run accessibility checker (e.g axeDevtools) on story book tabs page:
https://web-components.carbondesignsystem.com/?path=/story/components-tabs--default

### Suggested Severity

Medium (Severity 3) = User can complete task, and/or has a workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by Wenniea** [comment]
Hi, I can look into this issue!
